### DISCUSSION TITLE
How to do this elegantly without constructing a dynamica class?

### DISCUSSION THREAD

**Discussion by DM1-1** [tailwindlabs/tailwindcss#12743] [Help] [answered]
I'm well aware of why you say don’t construct class names dynamically. But I'd like to know your best practices for this question.

I want to set `line-clamp-${number}` by the param in Vue3. In other words, for a dynamically passed in number, it's not possible for me to generate a map for it.

```vue
<p :class="`blog-summary line-clamp-${summaryLineClamp}`">{{ summary }}</p>
```

Looking forward to your answer！Thank！

**Comment by wongjn** [comment, marked as answer]
You could consider using the `style` attribute:

```vue
<p
  :class="blog-summary overflow-hidden [display:-webkit-box] [-webkit-box-orient:vertical]"
  :style="{'-webkit-line-clamp': summaryLineClamp}"
>
```

**Reply by DM1-1** [reply]
Thank you! I learned that because of static analysis, dynamic class names really cannot be implemented in tailwindcss. 
<img width="235" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/88864535/614a2ddc-a3b8-45fc-a5be-dc06af58182c">
My approach is to use a configuration file to mark dynamic class names such as those that appear in the project, and configure it in tailwindcss.config
